<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerChart</class>
 <widget class="QDialog" name="PowerChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1184</width>
    <height>657</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1184</width>
    <height>657</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Power Graph</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/png/resources/icon.png</normaloff>:/png/resources/icon.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="title">
    <string>Left Chart</string>
   </property>
   <widget class="QRadioButton" name="radioButtonOutPwr_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonGridChgPwr_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Charge Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonShowGridInPwr_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPvPwr_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pv Power</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBatCap_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery Capacity</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>151</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>Right Chart</string>
   </property>
   <widget class="QRadioButton" name="radioButtonOutPwr_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Power</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonGridChgPwr_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Charge Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonShowGridInPwr_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPvPwr_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pv Power</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonBatCap_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery Capacity</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Settings</string>
   </property>
   <widget class="QComboBox" name="comboBoxFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxFixedChart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed Chart</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
